src/sysinfo.cc

```diff
@@ -21,9 +21,9 @@
 #include <stdlib.h>
 #include <string.h>
 #include <sys/resource.h>
+#include <sys/types.h>
```

Comment:
huh. that's frustrating given the google style guide around include ordering. Maybe you can also add a comment to explain that this ordering is important (and why) so good samaritans don't put it back :)